src/common/version.ts

```diff
@@ -49,15 +53,42 @@ export class Versions {
         return Versions.getValue(Constants.versionTempSensor, "1.0") as string;
     }
 
+    public static updateSystemModuleImageVersion(templateJson: any, moduleName: string, versionMap: Map<string, string>) {
+        if (templateJson) {
```

Comment:
are you checking if templateJson is undefined? Because of JS falsyness being vague, better to be explicit here.